content/doc/book/managing/system-configuration.adoc

```diff
@@ -11,5 +11,149 @@ ifdef::backend-html5[]
+at the top of your Jenkins home page.
+Use this page to display the name of the server
+and a short description of its purpose.
+It is also used to display announcements about scheduled down time
+and other information of interest to all Jenkins users.
+This field can include HTML tags
+or a pointer to a Wiki page and information about who to contact.
+
+Inline help is available for most fields on most *Manage Jenkins* pages.
+To access the help, select the *?* icon to the right of each field.
+Click the *?* icon again to hide the help text.
+
+== Executors on controller
+
+The *# of executors* field configures the number of executors on the controller.
+Typically, this should be set to 0 to prevent builds from running on the controller.
+We will discuss executors for nodes and agents in a later section.
+
+== Quiet period
+
+This field defines an interval to wait before actually starting a build.
+This can be used to collapse multiple adjacent commits into one build.
+
+////
+These fields no longer appear on the *System Configuration* page.
+== SCM tools
+
+This field configures tools on the controller itself;
+it does not control the tools that are configured in a build agent
+for running builds.
+
+Git is supported out-of-the-box.
+Plugin support is provided for
+Subversion, Mercurial, Perforce, Team Foundation Server, and other SCMs.
+In most cases, the plugin for the tool must be installed separately
+before you configure the tool on the System Configuration page.
+
+== SCM and authentication
+
+Most SCM tools support or require authentication when talking to servers.
+Different SCMs are configured differently,
+but most (including Git, SVN, Mercurial, and Team Foundation Server)
+use the Credentials plugin.
+However, Perforce requires that each project
+enters credentials separately.
+////
+
+== Jenkins location
+
+You must configure the **Jenkins Location** area
+to identify this particular instance of Jenkins.
+
+Two items must be defined here:
+
+* *Jenkins URL* -- Replace the `localhost:5000` string with information
+about the actual location people can use to access this Jenkins instance.
+This information can be:
+
+** The actual URL that people use to access Jenkins.
+** The host alias as defined by the name service.
+If your IT people cannot provide this, use dynamic DNS.
+This makes your instance easier for your users to access
+and makes your service relocatable.
+
+[NOTE]
+====
+*Invest in a good URL!*
+If your users cannot see Jenkins, much of the benefit is lost.
+Make the URL easy to  remember.  For example:
+
+* Bad: `\http://tjek-02.int.rev.example.org`
+* Good: `\https://jenkins.example.org`
+====
+
+* *System Admin e-mail address* -- Define the email address to which email
+about general Jenkins issues are sent.
+This can be an individual's email address
+or an email alias that contains multiple email addresses.
+
+== Timestamper
+
+The **Timestamper** fields define the format used
+to display timestamp information.
```

Comment:
```suggestion
* *System Admin e-mail address* -- Define the email address to which email about general Jenkins issues are sent.
This can be an individual's email address or an email alias that contains multiple email addresses.

== Timestamper

The **Timestamper** fields define the format used to display timestamp information.
```
small formatting suggestion